include/selene/util.h

```diff
@@ -91,7 +91,7 @@ inline int SetErrorHandler(lua_State *L) {
 }
 
 template<typename T, typename... Args>
-std::unique_ptr<T> make_unique(Args&&... args) {
+std::unique_ptr<T> sel::make_unique(Args&&... args) {
```

Comment:
It's my mistake. I use vc++ 2015, but it compile no error, So i did not see this one.
I will commit to fix it.